subsys/mpsl/fem/mpsl_fem.c

```diff
@@ -207,9 +182,14 @@ static int fem_nrf21540_gpio_configure(void)
 		},
 		.pa_pin_config = {
 #if DT_NODE_HAS_PROP(DT_NODELABEL(nrf_radio_fem), tx_en_gpios)
+			.cfg_type     = MPSL_FEM_PIN_CFG_TYPE_PIN_PORT,
 			.enable       = true,
 			.active_high  =
 				MPSL_FEM_GPIO_POLARITY_GET(tx_en_gpios),
+			.p_port       =
+				MPSL_FEM_GPIO_PORT_REG(tx_en_gpios),
+			.gpio_port_no =
+				MPSL_FEM_GPIO_PORT_NO(tx_en_gpios),
 			.gpio_pin     =
 				DT_GPIO_PIN(DT_NODELABEL(nrf_radio_fem),
```

Comment:
Creating a convenience macro
```c
#define MPSL_FEM_GPIO_PIN_NO(pin) DT_GPIO_PIN(DT_NODELABEL(nrf_radio_fem), pin)
```
for consistency would be a nice touch.

Reply:
Applied the suggestion.